### PR TITLE
support python libraries with `+` in the file name

### PR DESCRIPTION
**Pull request by levpachmanov**
- [x] All [tests](https://github.com/jfrog/build-info-go#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/build-info-go/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

### CODE REVIEW

**Comment by levpachmanov** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by levpachmanov** [comment]
Hi @yahavi @EyalDelarea can you please take a look?

**Review by sverdlov93 — COMMENTED** [review]
Thanks for your contribuiton! 
Please look at my inline comments

File: utils/pythonutils/utils.go [hunk 1]
```diff
@@ -307,5 +308,10 @@ func extractFileNameFromRegexCaptureGroup(pattern *gofrogcmd.CmdOutputPattern) (
 	if lastSlashIndex == -1 {
 		return filePath
 	}
-	return filePath[lastSlashIndex+1:]
+	latComponent := filePath[lastSlashIndex+1:]
```
[the remark this answers is not in the document]
Comment (levpachmanov) [reply utils/pythonutils/utils.go]: Done

File: utils/pythonutils/utils.go [hunk 2]
```diff
@@ -307,5 +308,10 @@ func extractFileNameFromRegexCaptureGroup(pattern *gofrogcmd.CmdOutputPattern) (
...
+	unescapedComponent, err := url.QueryUnescape(latComponent)
```
[the remark this answers is not in the document]
Comment (levpachmanov) [reply utils/pythonutils/utils.go]: Done

File: utils/pythonutils/utils.go [hunk 3]
```diff
@@ -307,5 +308,10 @@ func extractFileNameFromRegexCaptureGroup(pattern *gofrogcmd.CmdOutputPattern) (
...
+	if err != nil {
+		return latComponent
+	}
+	return unescapedComponent
```
[the remark this answers is not in the document]
Comment (levpachmanov) [reply utils/pythonutils/utils.go]: Done

**Comment by levpachmanov** [comment]
Hi @sverdlov93 , thank you for the comments. I fixed the code hopefully.

**Comment by levpachmanov** [comment]
Hi @sverdlov93, thank you for merging the pull request.
Any way I can help to get it released so the CLI can use it?

**Comment by sverdlov93** [comment]
Hi @levpachmanov 
JFrog CLI v2.56.0 https://github.com/jfrog/jfrog-cli/releases/tag/v2.56.0 is released!
Thanks again for your contribution!
Looking forward to your feedback on it.

**Pull request merged**